### PR TITLE
PLANET-5211 Home page Accessibility: Navigation bar and "Skip to..." links

### PR DESCRIPTION
**Pull request by dpivo**
Improved accessibility of Country List, Search Bar and Skip To Links.

### CODE REVIEW

File: assets/src/js/country_select.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const setupCountrySelect = function($) {
     let countries_html = $(
       '<div id="country-list" class="country-list">' +
       '<a class="international" href=""></a>' +
-      '<ul class="countries_list"></ul>' +
+      '<ul class="countries_list" aria-label="Worldwide site list" role="list"></ul>' +
```
Comment (mleray) [inline assets/src/js/country_select.js:28]: Don't we need to add translations for this? 🤔

File: assets/src/js/country_select.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const setupCountrySelect = function($) {
         let countries_sublist = $(
           '<li>' +
           '<h3 class="country-group-letter">' + index + '</h3>' +
-          '<ul class="countries_sublist"></ul>' +
+          '<ul class="countries_sublist" aria-label="Sites starting with the letter .' + index + '." role="list"></ul>' +
```
Comment (mleray) [inline assets/src/js/country_select.js:45]: Same question here, don't we need to add translations?

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Comment by dpivo** [comment]
**UAT**
Go to https://k8s.p4.greenpeace.org/test-titan/ 
Use screen reader to test navigation

**Pull request merged**